src/pages/docs/api/token-request-spec.mdx

```diff
@@ -0,0 +1,200 @@
+
+The receiving Ably server verifies the signature if present, the timestamp (which must be within 2 minutes of the current time), verifies that the nonce/timestamp combination has not been used previously, verifies that the requested validity period is permitted, and verifies that the requested capabilities are permitted for that token based on the key capabilities.
+
+The server may choose to subset the capabilities based on the capabilities of the key.
+
+The server replies with an access token, which is essentially a signed version of the resolved set of capabilities, plus other metadata associated with the token (such as expiry time).
+
+This access token can then be used for subsequent REST requests or Realtime connections. If a `clientId` was included in the request, then the token is associated with that `clientId`, and may be used to identify that client in operations that require identification (e.g. joining a channel that requires identification, or publishing a message with a verified `clientId`).
+
+### Parameter canonicalisation <a id="parameters" />
+
+The parameters of the Ably `TokenRequest` are normalized/canonicalized as follows:
+
+| Parameter | Description | Type |
+| --------- | ----------- | ---- |
+| keyName | No action required | String |
+| ttl | The decimal integer representation, without leading zeros, of the requested life of the Ably Token in milliseconds. If none is specified a default of 1 hour is used | Integer |
+| capability | This is a canonicalized representation of the channel paths and associated [operations](#capability-operations) in the capability. It is a JSON stringified value of a JavaScript object of the form:<br/><br/>`{"channel1": ["operation1a", "operation1b", ...], "channel2": ["operation2a", "operation2b", ...], ...}`<br/><br/>with the following constraints:<br/>• all white-space is removed<br/>• channels are listed in forward lexicographic order<br/>• operations are listed in forward lexicographic order<br/>• there is no trailing comma on any list of array or object elements<br/>• all strings are quoted and escaped as per the JSON standard<br/>• for channel paths, the wildcard character `*` has special meaning. When the channel path is exactly `*`, then all channels are matched. If however, a channel ends with `*`, then the path before the `*` is used to match all channels in that namespace. For example, the channel path `user:*` matches all channels in the `user` namespace such as `user:john` and `user:matt`. [Find out more about channel namespaces](/docs/channels#channel-namespaces)<br/>• for operations, the wildcard character `*` has special meaning. When the operation is exactly `*`, then [all operations](#capability-operations) are supported. Otherwise, a [permitted operation](#capability-operations) string must be provided | String |
```

Comment:
Optional formatting (for this phase).

A little line inbetween for spacing each bullet